modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/common_main.scss

```diff
@@ -99,6 +99,12 @@
 		}
 	}
 
+	.widget-mode-card {
+		.card {
+			max-width: 65%;
```

Comment:
This would be better if it was a fixed width value like 300px. It will still expand to be as wide as the container if it's something like 250px.